maryam/core/util/iris/topic.py

```diff
@@ -0,0 +1,225 @@
+# core/util/iris/topic.py
+# Hatma Suryotrisongko
+
+import pandas as pd
```

Comment:
imports are heavy and decrease the speed of the framework. So, try to import them in the class.
also, add them all to the requirements file.